src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,115 @@
+import { gray } from "d3-color";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import AsyncSelect from "react-select/async";
+
+const searchBarStyles = {
+  control: (provided) => ({
+    ...provided,
+    fontFamily: "Gotham-Book",
+    color: "white",
+    backgroundColor: gray,
+    fontSize: 16,
+    border: "0px",
+    borderRadius: "8px",
+    padding: "4px",
+  }),
+  option: (provided, state) => ({
+    ...provided,
+    fontFamily: "Gotham-Book",
+    fontSize: 14,
+    color: "black",
+  }),
+  menu: (styles) => ({
+    ...styles,
+    border: "0px",
+    borderRadius: "8px",
+  }),
+  input: (provided) => ({
+    ...provided,
+    color: "white",
+  }),
+  placeholder: (provided, { isFocused }) => ({
+    ...provided,
+    color: isFocused ? "gray" : "white",
+  }),
+};
+
+//search component for search page
+function Searchbar() {
+  const router = useRouter();
+  const [searchInput, setSearchInput] = useState("");
+  const [selectedValue, setSelectedValue] = useState(null);
+
+  //helper function to fetch from api
+  function fetchData(url) {
+    return new Promise((resolve, reject) => {
+      fetch(url)
+        .then((res) => {
+          if (!res.ok) {
+            reject(res);
+          }
+          resolve(res.json());
+        })
+        .catch(reject);
+    });
+  }
+
+  //helper functions to concat strings
+  function clanSearchUrl(input) {
+    return "/api/search?q=" + input + "&type=clan&limit=2";
```

Comment:
```suggestion
    return `/api/search?q=${input}"&type=clan&limit=2`;
```
use a template literal instead of concatenating with a string